catalog/spatial/registry/registry-rest-endpoint/src/main/java/org/codice/ddf/registry/rest/endpoint/report/RegistryReportBuilder.java

```diff
@@ -106,17 +106,16 @@ public String getErrorHtml(String errorMessage) throws IOException {
     private Map<String, Object> getSummaryMap(Metacard metacard) {
         Map<String, Object> summaryMap = new HashMap<>();
 
-        Map<String, String> summaryAttributes =
-                ImmutableMap.of(RegistryObjectMetacardType.ORGANIZATION_NAME,
-                        "Organization Name",
-                        Metacard.POINT_OF_CONTACT,
-                        "Point of Contact",
-                        RegistryObjectMetacardType.ORGANIZATION_ADDRESS,
-                        "Address",
-                        RegistryObjectMetacardType.ORGANIZATION_EMAIL,
-                        "Email Addresses",
-                        RegistryObjectMetacardType.ORGANIZATION_PHONE_NUMBER,
-                        "Phone Numbers");
+        Map<String, String> summaryAttributes = ImmutableMap.of(Contact.POINT_OF_CONTACT_NAME,
+                "Organization Name",
+                Metacard.POINT_OF_CONTACT,
```

Comment:
Should we still be setting the `Metacard.POINT_OF_CONTACT` in the `RegistryPackageConverter` now that we have `ContactAttributes`?